playbook-website/app/javascript/components/Website/src/components/Scaffold.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { useParams, useSearchParams, NavLink } from "react-router-dom"
+import {Title} from "playbook-ui"
+export default function Scaffold() {
+  const { name, type: pathType } = useParams()
+  const [searchParams] = useSearchParams()
+  const queryType = searchParams.get("type")
+
+  const finalType = pathType || queryType || "react"
+
+  const formatName = (name: any) => {
+    return name
+      .split("_")
+      .map((word: any) => word[0].toUpperCase() + word.slice(1))
+      .join(" ")
+  }
+
+  return (
+    <div className='pl_md'>
+      <NavLink to={`/beta/kits/${name}`}>
```

Comment:
```tsx
      `/beta/kits/${name}`
```

The `beta/kits` part of this link is used in other places. It might make sense to abstract this away to an import so that you can change it in one place instead of multiple.

Reply:
Worth considering. We will swap this URL with the live one at some point.